content/play/online.md

```diff
@@ -3,6 +3,4 @@ title: "Play Online"
 weight: 1500
 ---
 
-The easiest way to play Lc0 if you don't want to [install it on your computer]({{<ref "quickstart">}}) is to use one of **Lichess** bots.
-
-TODO(Adapt text from blog post or write a new one).
\ No newline at end of file
+The easiest way to play Lc0 if you don't want to [install it on your computer]({{<ref "quickstart">}}) is to use one of **Lichess** bots. You can find them in the respective <a><href="https://lichess.org/team/all-the-leelas">lichess team</a>. The ones that are almost always online are <a href="https://lichess.org/@/Lc0Test">Lc0Test</a>, <a href="https://lichess.org/@/leela2200">leela2200</a> or <a href="https://lichess.org/@/MiniHuman">MiniHuman</a>. There is also an official Leela bot called <a href="https://lichess.org/@/LeelaChess">LeelaChess</a>, but most of the time it's not online.  
```

Comment:
Thanks for the suggested change.

In markdown, the recommended way to insert a link is `[text](url)` rather than `<a href="url">text</a>`.

(optional) Also I believe (although not sure anymore) that unless there are two empty lines between lines, all text is combined into a single paragraph. If it's so, it may make sense to write all sentences on separate lines (wikipedia wiki style), to avoid extremely long lines.